saleor/static/dashboard-next/orders/views/OrderDetails/index.tsx

```diff
@@ -546,253 +451,213 @@ export const OrderDetails: React.StatelessComponent<OrderDetailsProps> = ({
+                                []
+                              ).map(country => ({
+                                code: country.code,
+                                label: country.country
+                              }))}
+                              errors={maybe(
+                                () => orderUpdate.opts.data.orderUpdate.errors,
+                                []
                               )}
+                              open={params.action === "edit-shipping-address"}
+                              variant="shipping"
+                              onClose={closeModal}
+                              onConfirm={variables =>
+                                orderUpdate.mutate({
+                                  id,
+                                  input: {
+                                    shippingAddress: {
+                                      ...variables,
+                                      country: variables.country.value
+                                    }
+                                  }
+                                })
+                              }
+                            />
+                            <OrderAddressEditDialog
+                              confirmButtonState={getMutationState(
+                                orderUpdate.opts.called,
+                                orderUpdate.opts.loading,
+                                maybe(
+                                  () => orderUpdate.opts.data.orderUpdate.errors
+                                )
+                              )}
+                              address={transformAddressToForm(
+                                maybe(() => order.billingAddress)
+                              )}
+                              countries={maybe(
+                                () => data.shop.countries,
+                                []
+                              ).map(country => ({
+                                code: country.code,
+                                label: country.country
+                              }))}
+                              errors={maybe(
+                                () => orderUpdate.opts.data.orderUpdate.errors,
+                                []
+                              )}
+                              open={params.action === "edit-billing-address"}
+                              variant="billing"
+                              onClose={closeModal}
+                              onConfirm={variables =>
+                                orderUpdate.mutate({
+                                  id,
+                                  input: {
+                                    billingAddress: {
+                                      ...variables,
+                                      country: variables.country.value
+                                    }
+                                  }
+                                })
+                              }
```

Comment:
This should be splited.

Reply:
That's not what this PR is about, and this view is waiting for the refactor anyway.